src/ext/java/org/opentripplanner/ext/transmodelapi/TransmodelGraphQLSchema.java

```diff
@@ -680,34 +697,34 @@ private GraphQLSchema create() {
 //                        .build())
 //                .build();
 
-//        GraphQLInputObjectType filterInputType = GraphQLInputObjectType.newInputObject()
-//                .name("InputFilters")
-//                .field(GraphQLInputObjectField.newInputObjectField()
-//                        .name("quays")
-//                        .description("Quays to include by id.")
-//                        .type(new GraphQLList(Scalars.GraphQLString))
-//                        .build())
-//                .field(GraphQLInputObjectField.newInputObjectField()
-//                        .name("lines")
-//                        .description("Lines to include by id.")
-//                        .type(new GraphQLList(Scalars.GraphQLString))
-//                        .build())
-//                .field(GraphQLInputObjectField.newInputObjectField()
-//                        .name("bikeRentalStations")
-//                        .description("Bike rentals to include by id.")
-//                        .type(new GraphQLList(Scalars.GraphQLString))
-//                        .build())
-//                .field(GraphQLInputObjectField.newInputObjectField()
-//                        .name("bikeParks")
-//                        .description("Bike parks to include by id.")
-//                        .type(new GraphQLList(Scalars.GraphQLString))
-//                        .build())
-//                .field(GraphQLInputObjectField.newInputObjectField()
-//                        .name("carParks")
-//                        .description("Car parks to include by id.")
-//                        .type(new GraphQLList(Scalars.GraphQLString))
-//                        .build())
-//                .build();
+        GraphQLInputObjectType filterInputType = GraphQLInputObjectType.newInputObject()
+                .name("InputFilters")
```

Comment:
I think we should rename this type to InputPlaceIds, the name here is bad it does not reflect what it is, but the roll it plays in the one situation it is used. The name is defined in a global space and the name is fare too generic.